minitwit/src/main/java/WebApplication.java

```diff
@@ -387,6 +396,116 @@ public static void updateLatest(Request request) {
         return "";
     };
 
+    public static Route serveSimFllws = (Request request, Response response) -> {
+
+        //update_latest(request); //TODO: call
+
+        var db = new SQLite();
+        var conn = db.getConnection();
+
+        var who_id = getUserID(db, request.params(":username"));
+
+        if (who_id == 0) {
+            conn.close();
+            response.status(404);
+            return false;
+        }
+
+        Gson gson = new Gson();
+        Fllws fllws = gson.fromJson(request.body(), Fllws.class);
+
+
+        if (Objects.equals(request.requestMethod(), "POST") && !fllws.follow.isEmpty()) {
+
+            var whom_id = getUserID(db, fllws.follow);
+
+            if (whom_id == 0) {
+                conn.close();
+                response.status(404);
+                return false;
+            }
+
+
+            var insert = conn.prepareStatement("insert into follower (\n" +
+                    "                who_id, whom_id) values (?, ?)");
+
+            insert.setInt(1, who_id);
+            insert.setInt(2, whom_id);
+            insert.execute();
+
+            conn.close();
+
+            response.status(200);
+            return true;
```

Comment:
This should be a 204 empty response. Instead of returning true you might need to return a empty string or NULL so the body is actually empty.